demo/admin/src/products/ManufacturerForm.gql.tsx

```diff
@@ -0,0 +1,62 @@
+// This file has been generated by comet admin-generator.
+// You may choose to use this file as scaffold by moving this file out of generated folder and removing this comment.
+import { gql } from "@apollo/client";
+
+export const manufacturerFormFragment = gql`
+    fragment ManufacturerFormDetails on Manufacturer {
+        address {
+            street
+            streetNumber
+            zip
+            country
+            alternativeAddress {
+                street
+                streetNumber
+                zip
+                country
+            }
+        }
+        addressAsEmbeddable {
+            street
+            streetNumber
+            zip
+            country
+            alternativeAddress {
+                street
+                streetNumber
+                zip
+                country
+            }
+        }
+    }
+`;
+export const manufacturerQuery = gql`
+    query Manufacturer($id: ID!) {
+        manufacturer(id: $id) {
+            id
+            updatedAt
```

Comment:
that's because I used the admin-generator to scaffold this file. and the admin-generator already did it this way. so I don't know what's the reason for this "best-practice"

Reply:
because it is not part of the form - the fragment only contains form fields